Add zenodo ref, add text, and write replies. (#59)
the-teachingRSE-project/end_of_RSEng: future_challenges_for_RSEng.tex

and community in the mid-term future, focusing on future challenges.
While future projections are common in industry~\cite{Gartner2025},
so far there is a projection lacking for the RSEng space.
We present here the results of a workshop discussion (notes content is archived here \cite{goth2025_WSPad}) at the
deRSE25 conference in Karlsruhe\footnote{deRSE25, session ``The End of {RSEng}?
Challenges and Risks for {RSEng}'': \url{https://events.hifis.net/event/1741/contributions/14026/}},
without claiming to represent the entire deRSE community.
The material presented here is based on the output of the workshop participants, but extended with the authors' ideas.

> Since deRSE25 was co-located with SE25, the annual conference for the Software Engineering community in Germany, the discussion was also open to software engineers,
> but a precise statistic of the workshop participants background was not taken.

Reviewer: ```suggestion
Since deRSE25 was co-located with SE25, the annual conference for the Software Engineering community in Germany, the discussion was also open to software engineers,
but no list or profile of the participants is available.
```

%FG-NOTE: I think we can use this part to wrap things a bit up, and describe how this future then looks like.
%GC-NOTE: Shall we add another story box here? Ideas what we write here?
In the previous sections, we have tried to list some of the trends that we foresee.

> The material presented here is based on the outputs of a workshop, but extended with the authors' ideas.

Reviewer: Not originating in this PR, but I guess we should bring the "extended with the authors' ideas" earlier, if not already there.
